pkg/controller/registry/resolver/resolver.go

```diff
@@ -96,7 +96,7 @@ func (r *OperatorsV1alpha1Resolver) ResolveSteps(namespace string, sourceQuerier
 		if op.Bundle() != nil {
 			bundleSteps, err := NewStepResourceFromBundle(op.Bundle(), namespace, op.Replaces(), op.SourceInfo().Catalog.Name, op.SourceInfo().Catalog.Namespace)
 			if err != nil {
-				return nil, nil, fmt.Errorf("failed to turn bundle into steps")
+				return nil, nil, fmt.Errorf("failed to turn bundle into steps %s", err.Error())
```

Comment:
It would be nice to add a colon to separate the generic error message from actual error message: `return nil, nil, fmt.Errorf("failed to turn bundle into steps: %s", err.Error())`.

Reply:
done